PixelHunter1995/WalkingAreaLib/WalkingArea.cs

```diff
@@ -26,7 +29,12 @@ public override string ToString()
 
         public void Draw(GraphicsDeviceManager graphics, SpriteBatch spriteBatch, double scaling)
         {
-            partition.Draw(graphics, spriteBatch, scaling);
+            if (!GlobalSettings.Instance.Debug)
```

Comment:
Since you moved this, we will draw PolygonDog polygons even when debug isn't true, which I don't think is intended.

Reply:
Fixed